src/main/views/iac-daily-list.njk

```diff
@@ -24,7 +24,7 @@
         {% set venueContact = listData['venue']['venueContact'] %}
         <p>{{ importantInformationP1 }}</p>
         <p>{{ importantInformationP2 }}</p>
-        <p>{{ venueContact["venueEmail"] }} {{ venueContact["venueTelephone"] }}</p>
+        <p>{{ venueContact["contactia@justice.gov.uk"] }} {{ venueContact["03001231711"] }}</p>
```

Comment:
There's a bug here - the values are being read from venueContact[]. Instead they need to be the value on its own with no venueContact as they are now the displayed values, rather than being rad from the schema

Reply:
Hardcoded contact details. Thanks Chris.